config/locales/en/unwithdraw/unwithdraw.yml

```diff
@@ -2,3 +2,10 @@ en:
   unwithdraw:
     no_managing_editor_permission:
       title: Only a managing editor can undo withdraw
+    confirm:
+      title: Undo the withdrawal of '%{title}'
+      description_govspeak: |
+        Are you sure you want to undo the withdrawal?
+
+        This will remove the withdrawn banner and republish
+        the content immediately. It will not send an email alert to users.
```

Comment:
For consistency this should be in `config/locales/en/unwithdraw/confirm.yml` where I understand our convention is we have locale files ordered via /controller/action.yml

Reply:
Fixed